packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1364,14 +1372,8 @@ export function markRenderEventTimeAndConfig(
     workInProgressRootLatestProcessedExpirationTime = expirationTime;
   }
   if (suspenseConfig !== null) {
-    if (
-      expirationTime < workInProgressRootLatestSuspenseTimeout &&
-      expirationTime > Idle
```

Comment:
This condition seems dropped on the floor, why?

Reply:
Yeah I would guess this branch needs to stay as it was except replace `workInProgressRootSuspenseLatestTimeout` with `workInProgressRootCanSuspendUsingConfig.timeoutMs`.